bluepyopt/deapext/optimisations.py

```diff
@@ -268,7 +269,32 @@ def run(self,
             offspring_size = self.offspring_size
 
         # Generate the population object
-        pop = self.toolbox.population(n=offspring_size)
+        if population is not None:
+
+            if len(population) != offspring_size:
```

Comment:
I think we might have to discuss here what we mean with 'population'. I'd say population = parents + offspring. Don't know how far that makes sense here. Should we maybe consider renaming the field people can specify 'offspring' ?

Reply:
Ok !